pkg/promclient/promclient.go

```diff
@@ -589,7 +591,7 @@ func (c *Client) AlertmanagerAlerts(ctx context.Context, base *url.URL) ([]*mode
 	span, ctx := tracing.StartSpan(ctx, "/alertmanager_alerts HTTP[client]")
 	defer span.Finish()
 
-	body, _, err := c.req2xx(ctx, &u, http.MethodGet)
+	body, _, err := c.req2xx(ctx, &u, http.MethodGet, map[string]string{})
```

Comment:
Use a pointer in `req2xx` and pass nil
```suggestion
	body, _, err := c.req2xx(ctx, &u, http.MethodGet, nil)
```